src/CoreBundle/Resources/translations/admin.en.xliff

```diff
@@ -1052,38 +1052,60 @@
                 <source>chameleon_system_core.cms_module_page_tree.headline</source>
                 <target>Navigation Administration</target>
             </trans-unit>
+
+            <!-- deprecated since 3.3.8 -->
```

Comment:
3.3.8 ?

Should that be a note as in the admin.de.xliff?

Reply:
ok

Reply:
ok